pkg/util/export/merge.go

```diff
@@ -67,7 +67,7 @@ const defaultMaxFileSize = 32 * mpool.MB
 //
 // - Merge.doMergeFiles handle one job flow: read each file, merge in cache, write into file.
 type Merge struct {
-	task        task.Task               // set by WithTask
+	task        task.AsyncTask          // set by WithTask
```

Comment:
i see. 
changes what i got in pr.
1. old type `task.Task` rename to `task.AsyncTask`
2. new interface named `task.Task`